growthbook/growthbook
# 904 Remove deleted references from Slack Integrations

### Features and Changes

When tags, projects, and environments are deleted, the references need to be removed from the Slack Integration collection. These changes remove those references.

- Closes #904 

### Dependencies

n/a

### Testing

1. Create some tags you're ok with deleting
2. Create some projects you're ok with deleting
3. Create some environments you're ok with deleting
4. Create a Slack integration and add the above tags, projects and environments as filters
5. Go back to those referenced resources (tags, projects, environments) and delete them one at a time.

You should see the Slack integrations get updated thanks to the auto-refreshing logic, i.e. the deleted resources should have their references removed from all applicable Slack integrations.
